sci_analysis/graphs/vector.py

```diff
@@ -262,19 +262,31 @@ def __init__(self, xdata, ydata=None, **kwargs):
         :param _points: Display the scatter points.
         :param _contours: Display the density contours
         :param _boxplot_borders: Display the boxplot borders
+        :param _highlight: an array-like with points to highlight based on labels
+        :param _labels: a vector object with the graph labels
         :param _title: The title of the graph.
         :param _save_to: Save the graph to the specified path.
         :return: pass
         """
         self._fit = kwargs['fit'] if 'fit' in kwargs else True
         self._points = kwargs['points'] if 'points' in kwargs else True
+        self._labels = kwargs['labels'] if 'labels' in kwargs else [False]
+        self._highlight = kwargs['highlight'] if 'highlight' in kwargs else None
         self._contours = kwargs['contours'] if 'contours' in kwargs else False
         self._contour_props = (31, 1.1)
         self._boxplot_borders = kwargs['boxplot_borders'] if 'boxplot_borders' in kwargs else False
         self._title = kwargs['title'] if 'title' in kwargs else 'Bivariate'
         self._save_to = kwargs['save_to'] if 'save_to' in kwargs else None
         yname = kwargs['yname'] if 'yname' in kwargs else 'y Data'
         xname = kwargs['xname'] if 'xname' in kwargs else 'x Data'
+        if 'labels' in kwargs:
+            if len(kwargs['labels']) != len(xdata):
+                raise AttributeError('The length of passed labels does not match the length of xdata and ydata')
+            if ndarray.any(kwargs['labels'].index != xdata.data.index):
+                raise AttributeError('The index of passed labels does not match the index of xdata and ydata')
+        if 'highlight' in kwargs:
```

Comment:
Both of these if statements can be combined into a single statement.

Reply:
Combined these in the latest update